### DISCUSSION TITLE
SSL certificate issue

### DISCUSSION THREAD

**Discussion by stormyzio** [vercel/community#3568] [Help]
### Summary

# Hello!

I created a reactjs website on Vercel with a redirection to my personal domain name ( stormyz.xyz ). To avoid the site insecure message, I added the CAA ( 0 issue "letsencrypt.org" ), but still doesn’t work after 48 hours. 
If anyone could help me that would be great.

Thank you!

### Example

https://stormyz.xyz

### Steps to Reproduce

I created a website, linked my domain name, hosted by OVH, to vercel ( A stormyz.xyz to kiube.vercel.app ), added the correct CAA ( 0 issue "letsencrypt.org" ).

**Comment by smaeda-ks** [comment]
looks like it's working now

**Reply by stormyzio** [reply]
Hello, Thanks!
Yes it's working, I asked Google and they accepted.
